project/plugins.sbt

```diff
@@ -3,11 +3,11 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
   Resolver.ivyStylePatterns
 )
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.15.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"    % "2.2.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.18")
-addSbtPlugin("org.scalameta"     % "sbt-scalafmt"          % "2.4.6")
```

Comment:
🤔 This doesn't resolve any versions, it's just formatting of the plugins file.

Reply:
The addition of the second % character ensures that the scala version of the plugin is the same as that defined in build.sbt